common/kafka/stats_enum.h

```diff
@@ -18,47 +18,19 @@
 #include <vector>
 #include "common/stats/stats.h"
 
-#define STATS_ADD_METRIC common::Stats::get()->AddMetric
-#define STATS_INCR common::Stats::get()->Incr
+const std::string kKafkaConsumerSeek = "kafka_consumer_seek";
```

Comment:
It's not a good practice to define std::string object in header files.
The reason and a solution can be found here: https://stackoverflow.com/a/10201929/4959149

Reply:
Thanks for pointing this out, will fix it.